distribution/docs/src/main/jdocs/content/_developingComponents/developing-endpoints-contents.adoc

```diff
@@ -0,0 +1,59 @@
+:title: Developing Endpoints
+:type: developingComponent
+:status: published
+:link: _developing_endpoints
+:summary: Creating a custom endpoint
+:implements: https://www.w3.org/2001/sw/wiki/REST[REST specification].
+
+Custom endpoints can be created, if necessary.
+See <<_available_endpoints,Available Endpoints>> for descriptions of provided endpoints.
+
+Complete the following procedure to create an endpoint. 
+
+. Create a Java class that implements the endpoint's business logic. Example: Creating a web service that external clients can invoke.
+
+. Add the endpoint's business logic, invoking `CatalogFramework` calls as needed.  
+
+. Import the ${branding} packages to the bundle's manifest for run-time (in addition to any other required packages): +
+`Import-Package: ddf.catalog, ddf.catalog.*`
+
+. Retrieve an instance of `CatalogFramework` from the OSGi registry. (Refer to <<_working_with_osgi_-_service_registry,Working with OSGi - Service Registry>> for examples.)
+
+Deploy the packaged service to ${branding}.
```

Comment:
This step doesn't show up in the list
![image](https://cloud.githubusercontent.com/assets/8041246/25190052/f62f5114-24df-11e7-9c73-59b1751ffb9b.png)